### PR TITLE
Better file structure

### PR DESCRIPTION
**Pull request by dancork**
## What does this PR do?
I've been playing with jekyll lately and think that this is a better structure. All html is in `_layouts` so all content is markdown :)

This means anyone using prose.io can edit all content on the site.

## How to test
- Checkout branch
- `jekyl serve _config.yml,_config_dev.yml`
- Visit `http://localhost:4000/`
- Make sure all pages still work

#### Quality checklist (for PR author to complete BEFORE code review):
- [x] I've updated documentation with any changes to procedures..
- [x] I am ready for this to be code reviewed, merged and tested.

#### IP or Developer Review:
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve checked for coding anti-patterns.
- [x] +1 from me!

#### Software Engineer, Software Architect or Developer review:
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve checked for coding anti-patterns.
- [x] +1 from me!

### CODE REVIEW

**Comment by LouisPryer** [comment]
Taking IP/Dev

**Comment by LouisPryer** [comment]
:+1:

File: speaking/index.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+layout: speaking
+title: Speaking Geek
```
Comment (jodiedoubleday) [inline speaking/index.md]: Not to do with this PR but I'm not a fan of this name, i's prefer just 'speaking'

**Comment by jodiedoubleday** [comment]
This is a really good addition/change happy for this to go live :) :+1:

File: speaking/index.md (hunk 1 above)
Comment (dancork) [inline speaking/index.md]: I'll update it anyway :)

**Pull request merged**